storage/innobase/log/log0online.cc

```diff
@@ -1051,7 +1051,13 @@ void log_online_read_shutdown(void) noexcept {
 }
 
 /** Shut down the constant part of the log tracking subsystem */
-void log_online_shutdown(void) noexcept { mutex_free(&log_bmp_sys_mutex); }
+void log_online_shutdown(void) noexcept {
+  /* log_online_init() is called after buf_pool_init(), and if buf_pool_ptr !=
+   * nullptr, then log_bmp_sys_mutex was created */
+  if (!buf_pool_ptr) return;
+
```

Comment:
Its better to use a log online state (or a corresponding global variable) instead of buf_pool